frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/create-local-volume-set-step/create-local-volume-set-step.tsx

```diff
@@ -221,6 +225,9 @@ export const CreateLocalVolumeSet: React.FC<CreateLocalVolumeSetProps> = ({
                 storageClassName={storageClass.name}
                 allNodes={allNodes.current}
                 nodes={nodes}
+                defaultVolumeMode={
+                  isMCG ? diskModeDropdownItems.FILESYSTEM : diskModeDropdownItems.BLOCK
```

Comment:
For mcg default volume mode - filesystem,
For full deployment, volume mode - block